integration/yarn/src/main/java/tachyon/yarn/ApplicationMaster.java

```diff
@@ -178,8 +179,12 @@ public void requestContainers() throws Exception {
     String[] nodes = {mMasterAddress};
 
     // Make container request for Tachyon master to ResourceManager
+    boolean relaxLocality = true;
+    if (StringUtils.isNotEmpty(mMasterAddress) && !mMasterAddress.equals("localhost")) {
+      relaxLocality = false;
+    }
```

Comment:
Nice. I see TACHYON-1165 has fixed that. So, I will just remove the isEmpty code.